ContuityTests/ModelsTests/JotTests.swift

```diff
@@ -28,4 +33,44 @@ class JotTests: XCTestCase {
         XCTAssertEqual(statement?.columnCount, 7)
         XCTAssertEqual(statement?.columnNames, ["id", "data", "queue", "createdAt", "modifiedAt", "latitude", "longitude"])
     }
+    
+    func testRead() {
+        self.freshTables()
+        let text1: String = "Whoa, I can't believe this works"
+        let text2: String = "The text for jot2"
+        let text3: String = "Scuba diving with the squad #funtimes"
+        let now = "now"
+        let jot1 = Jot(id: 4, data: text1, queue: false, createdAt: now, modifiedAt: now, latitude: 0, longitude: 0)
+        jot1.write()
+        let jot2 = Jot(id: 2, data: text2, queue: false, createdAt: now, modifiedAt: now, latitude: 0, longitude: 0)
+        jot2.write()
+        let createdAt3 = "04-20-6969 10:55:30"
+        let jot3 = Jot(id: 3, data: text3, queue: true, createdAt: createdAt3, modifiedAt: createdAt3, latitude: 0, longitude: 0)
+        jot3.write()
+        
+        XCTAssertNotEqual(jot1, jot2)
+        let readJot = Jot.read(givenID: 4)
+        XCTAssertEqual(jot1, readJot)
+        XCTAssertNotEqual(jot2, readJot)
+        XCTAssertEqual(jot3, Jot.read(givenID: 3))
```

Comment:
not really an issue but you did `let readJot = Jot.read(givenID: 4)` for one assert and then did it inline for another. Would be nicer to be consistent

Reply:
Also I'm not really sure what this is testing? Is it different from the previous asserts other than it being a different jot?